docker/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:20.04
+
+ENV TESTING_HOME=/testing
+
+ENV GCC_HOME=$TESTING_HOME/gcc
+ARG GCC_REPO
+ENV GCC_REPO=${GCC_REPO:-git://gcc.gnu.org/git/gcc.git}
+ARG GCC_VERSION
+ENV GCC_VERSION=${GCC_VERSION:-master}
+
+ENV LLVM_HOME=$TESTING_HOME/llvm
+ARG LLVM_REPO
+ENV LLVM_REPO=${LLVM_REPO:-https://github.com/llvm/llvm-project.git}
+ARG LLVM_VERSION
+ENV LLVM_VERSION=${LLVM_VERSION:-main}
+
+ARG SDE_REPO
+ENV SDE_REPO=${SDE_REPO:-https://software.intel.com/content/dam/develop/external/us/en/documents/downloads/}
+ARG SDE_VERSION
+ENV SDE_VERSION=${SDE_VERSION:-sde-external-8.69.1-2021-07-18-lin}
+
+ENV YARPGEN_HOME=$TESTING_HOME/yarpgen
+ARG YARPGEN_REPO
+ENV YARPGEN_REPO=${YARPGEN_REPO:-https://github.com/intel/yarpgen}
+ARG YARPGEN_VERSION
+ENV YARPGEN_VERSION=${YARPGEN_VERSION:-main}
+
+RUN apt-get -y update &&\
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    cmake \
+    curl \
+    flex \
+    gcc \
+    gcc-multilib \
+    g++ \
+    git \
+    ninja-build \
+    python3 \
+    wget
+# No need to do cleanup, ubuntu container will do it for us
+
+WORKDIR $TESTING_HOME
+
+COPY build-gcc.sh $TESTING_HOME/
+RUN ["/bin/bash", "-c", "./build-gcc.sh"]
```

Comment:
1. This look like complication, just have `RUN ./build-gcc.sh`, which will be sufficient.
2. Start `./build-gcc.sh` from `#!/bin/bash -e` (note `-e`, this way errors will be reported back to calling context for every command, not only for the last one in the script), see example [here](https://github.com/ispc/ispc/blob/main/.github/workflows/scripts/build-ispc.sh#L1)

Reply:
Fixed